pyeudiw/jwt/__init__.py

```diff
@@ -38,23 +44,38 @@
     "EC": "A256GCM"
 }
 
+KeyLike = ECKey | RSAKey | OKPKey | SYMKey
+SerializationFormat = Literal["compact", "json"]
 
-class JWEHelper():
-    """
-    The helper class for work with JWEs.
-    """
 
-    def __init__(self, jwk: Union[JWK, dict]):
+class JWHelperInterface:
```

Comment:
all these classes must be moved in filenames with their names and `__init__.py` must import them automatically on package load, as they would not have been divided into different files

@LadyCodesItBetter 

we want this rule to be applied to all the packages. 

Huge and verbose `__init__.py` files are not appropriate for this project

Reply:
We do not do this anymore, this task will be done in PR https://github.com/italia/eudi-wallet-it-python/pull/305